packages/cli-plugin-scaffold-react-component/CHANGELOG.md

```diff
@@ -0,0 +1,93 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
```

Comment:
Remove this file. All of the content below is irrelevant to the newly added package.